src/util.jl

```diff
@@ -198,8 +198,7 @@ function getlineend(str, i=1, l=lastindex(str))
         y = iterate(str, i)
     end
 
-    # TODO Is this correct?
-    return i-1
+    return prevind(str, i)
```

Comment:
Well I think this is the same. `i-1` seems more correct. If a line ends in a greek letter `prevind` is wrong because it will land on the first byte of the greek character.

Reply:
No, I think here the PR is actually correct. If you look at where the return value of this is used, it is all cases that expect the byte index of the last character before the line break. [This](https://github.com/JuliaComputing/TextParse.jl/blob/68d35e91c9449e422d6a3f85dab6b838046a593f/src/csv.jl#L201), [this](https://github.com/JuliaComputing/TextParse.jl/blob/68d35e91c9449e422d6a3f85dab6b838046a593f/src/csv.jl#L459) and I think [this also](https://github.com/JuliaComputing/TextParse.jl/blob/68d35e91c9449e422d6a3f85dab6b838046a593f/src/csv.jl#L487) expects the byte index index of the last character.